packages/react/src/components/Storage/FileUploader/displayText.ts

```diff
@@ -0,0 +1,35 @@
+export const defaultFileUploaderDisplayText = {
+  filesUploadedText(count: number): string {
+    return `${count} ${count === 1 ? 'file uploaded' : 'files uploaded'}`;
+  },
+  remainingFilesText(count: number): string {
+    return `${count} ${count === 1 ? 'file' : 'files'} selected`;
+  },
+  uploadingText(percentage: number): string {
+    return `Uploading${percentage > 0 ? `: ${percentage}%` : ''}`;
+  },
+  uploadButtonText(count: number): string {
+    return `Upload ${count} ${count === 1 ? 'file' : 'files'}`;
+  },
+  maxFilesErrorText(count: number): string {
+    return `Cannot choose more than ${count} ${
+      count === 1 ? 'file' : 'files'
+    }. Remove files before updating`;
+  },
+  errorText(message: string): string {
+    return message;
+  },
```

Comment:
Sorry to bring this up so late, but wondering if we should prefix the functions with `get` (`getErrorText`). Would be easier to discern which of these values are just strings and which are functions that return a string. What do you think?

Reply:
Makes sense to me

Reply:
Done!